src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Styled from './VideoCard.styles';
+
+const VideoCard = ({ thumbnail, title, body }) => (
+  <Styled.Container>
+    <Styled.Preview src={thumbnail} />
+    <Styled.Title>{title}</Styled.Title>
+    <Styled.Body>{body}</Styled.Body>
+  </Styled.Container>
+);
```

Comment:
I like how you defined the `VideoCard`, here you can change the way you export the components from the `VideoCard.styles` and change the import to something like:

```jsx
import { Container, Preview, Title, Body } from './VideoCard.styles';
```

and if you import the components this way you can use them just with the name of the component, like `<Container> ... </Container>`, you don't have to write `Styled.` everytime